src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/SocketOutput.cs

```diff
@@ -673,6 +673,7 @@ private void DoDisconnectIfNeeded()
                 Self._socket.Dispose();
                 Self._connection.OnSocketClosed();
                 Self._log.ConnectionStop(Self._connectionId);
+                KestrelEventSource.Log.ConnectionStop(Self._connection);
```

Comment:
🔔 @halter73. Is this the right place to put this? We need to ensure this is _always_ be called for any connection successfully opened.